receiver/azuremonitorreceiver/README.md

```diff
@@ -25,7 +25,7 @@ The following settings are required:
 The following settings are optional:
 
 - `auth.authenticator`: Specifies the component ID to use to authenticate requests to Azure API. Use [azureauth extension](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/azureauthextension).
-- `credentials` (Deprecated [v0.126.0]: use `auth` instead)(default = service_principal): Specifies the used authentication method. Supported values are `service_principal`, `workload_identity`, `managed_identity`, `default_credentials`.
+- `credentials` (Deprecated [v0.126.0 without batch, v0.129.0 with batch api]: use `auth` instead)(default = service_principal): Specifies the used authentication method. Supported values are `service_principal`, `workload_identity`, `managed_identity`, `default_credentials`.
```

Comment:
You can say it's deprecated since v0.129.0 instead.

Reply:
Fixed.